source/draw/gpu/opengl/OpenGLPacked.ooc

```diff
@@ -84,7 +85,7 @@ OpenGLPacked: abstract class extends GpuImage {
 					gpuMap add("texture0", image)
 			}
 		}
-		gpuMap useProgram(this, projection * view * model, textureTransform)
+		gpuMap useProgram(this, flat * projection * view * model, textureTransform)
```

Comment:
Since reference coordinates goes from -half size to +half size and they need to go from -1 to +1, we simply divide by (inputSize * 0.5). If the offset is in read coordinates then we take the inverse of the read transfrom as the write transform.